services/post_service.py

```
@@ -0,0 +1,85 @@
+
+from fastapi import HTTPException,status
+from fastapi.params import Depends
+
+from repositories.post_repository import PostRepository
+from repositories.user_repository import UserRepository
+from schemas.post import PostsCreateRequest, PostsUpdateRequest
+
+
+class PostService:
+    def __init__(self,post_repo:Annotated[PostRepository, Depends(PostRepository)],
+                      user_repo:Annotated[UserRepository,Depends()]) -> None:
```

Ruff (0.14.11)

Unused method argument: `user_repo`

(ARG002)